code/game/objects/items/devices/uplink.dm

```diff
@@ -387,21 +387,33 @@ A list of items and costs is stored under the datum of every game mode, alongsid
 	discord_bot.send_to_cciaa("Announcer - Fake announcement:`[title]` - `[message]`, sent by [user]!")
 	qdel(src)
 
-//ninja
-/obj/item/device/ninja_uplink
-	name = "infiltrator uplink"
-	desc = "A small device used for access to a restricted cache of specialized items."
+/obj/item/device/special_uplink
+	name = "special uplink"
+	desc = "A small device used to access a restricted cache of specialized items."
```

Comment:
Should probs rename that desc to something that describes only it´s external appearence and put the rest in antag_desc